integrated_channels/degreed2/client.py

```diff
@@ -674,11 +674,12 @@ def _get_oauth_access_token(self, scope):
         """
         config = self.enterprise_configuration
         url = self.get_oauth_url()
+        use_encrypted_user_data = getattr(settings, 'FEATURES', {}).get('USE_ENCRYPTED_USER_DATA', False)
         data = {
             'grant_type': 'client_credentials',
             'scope': scope,
-            'client_id': config.client_id,
-            'client_secret': config.client_secret,
+            'client_id': config.decrypted_client_id if use_encrypted_user_data else config.client_id,
+            'client_secret': config.decrypted_client_secret if use_encrypted_user_data else config.client_secret,
```

Comment:
Make sure to disable the value here https://github.com/search?q=repo%3Aedx%2Fedx-internal%20use_encrypted_user_data&type=code before merging

Reply:
Should we replace the two fields with encrypted ones [here](https://github.com/openedx/edx-enterprise/blob/master/integrated_channels/degreed2/models.py#L79) as well or are we going to do that in the follow up PR?

Reply:
> Make sure to disable the value here https://github.com/search?q=repo%3Aedx%2Fedx-internal%20use_encrypted_user_data&type=code before merging

[Unset USE_ENCRYPTED_USER_DATA to true in prod configuration ](https://github.com/edx/edx-internal/pull/10601) i have updated it here.

Reply:
> Should we replace the two fields with encrypted ones [here](https://github.com/openedx/edx-enterprise/blob/master/integrated_channels/degreed2/models.py#L79) as well or are we going to do that in the follow up PR?

[Remove degreed unencrypted columns Ent8009](https://github.com/openedx/edx-enterprise/pull/2069) it is going to be replaced in this follow up PR